cpp/include/ray/api/ray_config.h

```diff
@@ -52,6 +47,26 @@ class RayConfig {
 
   // A mapping the names of custom resources to the quantities for them available.
   std::unordered_map<std::string, int> resources;
+
+  // Boolean flag indicating whether or not to start the Ray dashboard, which displays the
+  // status of the Ray cluster. If this argument is None, then the UI will be started if
+  // the relevant dependencies are present.
+  bool include_dashboard = false;
+
+  // The host to bind the dashboard server to. Can either be localhost (127.0.0.1) or
+  // 0.0.0.0 (available from all interfaces). By default, this is set to localhost to
+  // prevent access from external machines.
+  std::string dashboard_host = "127.0.0.1";
+
+  // The port to bind the dashboard server to.
+  // Defaults to 8265 and Ray will automatically find a free port if
+  // 8265 is not available.
+  int32_t dashboard_port = -1;
```

Comment:
if  use the default port  -1 what will happen?

Reply:
I have a "if" here https://github.com/ray-project/ray/pull/18709/files#diff-5d865b359cdf9da01de72d3dbb9ecd883d2df7f9492ddcee0d95d6632c9cc678R70